### PR TITLE
[Space] fix for if copyright key is none

### PR DESCRIPTION
**Pull request by owocado**
also fuck nasa.

### CODE REVIEW

File: space/core.py [hunk 1]
```diff
@@ -114,6 +114,11 @@ async def apod_text(self, data: dict, context: Union[commands.Context, discord.T
             return "Astronomy Picture of the Day: `Impossible to get Nasa API.`"
 
         details = data["explanation"]
+        try:
+            credits = data["copyright"]
+        except KeyError:
+            credits = None
+            pass
```
Comment (PredaaA) [inline space/core.py]:
```suggestion
```

File: space/core.py [hunk 2]
```diff
@@ -131,7 +136,7 @@ async def apod_text(self, data: dict, context: Union[commands.Context, discord.T
                 icon_url="https://i.imgur.com/Wh8jY9c.png",
             )
             em.set_image(url=data["url"])
-            em.set_footer(text="Image Credits: {} • Today is {}".format(data["copyright"], data["date"]))
+            em.set_footer(text="Image Credits: {} • Today is {}".format(credits, data["date"]))
```
Comment (PredaaA) [inline space/core.py]:
```suggestion
            em.set_footer(text="Image Credits: {} • Today is {}".format(data.get("copyright", "Unknown"), data["date"]))
```

**Review by PredaaA — CHANGES_REQUESTED** [review]
A `.get` would be better here. Also "Unknown" instead of "None" may be more user friendly.

**Comment by owocado** [comment]
~~Hi @PredaaA ,~~

~~the reason I didn't use `.get()` is because it would make it inconsistent with rest of the format on how data is fetched from API, because at all places the `data[""]` format is used. Thats why. :P~~

Pls ignore this and see below commit kthx. Thank you Preda. <3

**Review by PredaaA — APPROVED** [review]

**Pull request merged**